app/assets/stylesheets/publisher.scss

```diff
@@ -1,6 +1,6 @@
 .publisher {
   z-index: 1;
-  color: $text-grey;
+  color: #999;
```

Comment:
Color literals like `#999` should only be used in variable declarations; they should be referred to via variable everywhere else.